src/main/java/dev/symphony/harmony/mixin/combat/tridents_return_from_void/TridentEntityMixin.java

```diff
@@ -22,10 +23,12 @@ protected TridentEntityMixin(EntityType<? extends ProjectileEntity> entityType,
 
     @Inject(method = "tick", at = @At("HEAD"))
     private void tickMixin(CallbackInfo ci) {
-        if (!this.getWorld().isClient) {
-            if (!this.dealtDamage && this.getBlockPos().getY() < 1 && this.getWorld().getDimensionEntry().getKey().get().getValue().toString().contains("minecraft:the_end")) {
-                this.dealtDamage = true;
-                this.returnTimer = 0;
+        if(HarmonyConfig.tridentsReturnFromVoid){
+            if (!this.getWorld().isClient) {
+                if (!this.dealtDamage && this.getBlockPos().getY() < 1 && this.getWorld().getDimensionEntry().getKey().get().getValue().toString().contains("minecraft:the_end")) {
```

Comment:
This long chain of calls to find the end dimension can be replaced by `this.getWorld().getRegistryKey() == World.END`. Other than that, looks good to me!